### PR TITLE
fix(paramount): fixed presence

### PR DESCRIPTION
**Pull request by calicokatsu**
## Description 
- Escaped `if-else hell` (kinda)
- Fixed Live TV Presence
- Fixed Series Presence
- Series Presence now has the Episode Thumbnail as it's Large Image Key

## Acknowledgements
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`

## Screenshots
image
image

<details>
<summary> Series' have been reported to not be working. This push fixes that and the Series presence now also displays the episode thumbnail alongside the Title, Season and Episode. Along with fixing the Live TV Presence</summary>

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
Change title to: fix(Paramount): fixed presence

File: websites/P/Paramount+/presence.ts [hunk 1]
```diff
@@ -22,132 +22,158 @@ presence.on("UpdateData", async () => {
...
-		);
-		if (vidArea) {
-			if (path.includes("/movies")) movTitle = jsonData.name;
-			else if (path.includes("/video")) {
-				title = jsonData.partOfSeries.name;
-
-				seasonEpi =
-					`S${jsonData.partOfSeason.seasonNumber}:` +
-					`E${jsonData.episodeNumber} ${jsonData.name}`;
+		{ pathname: path } = window.location;
```
Comment (DarkVillager) [inline websites/P/Paramount+/presence.ts:25]: We prefer document.location
```suggestion
		{ pathname: path } = document.location;
```

File: websites/P/Paramount+/presence.ts [hunk 2]
```diff
@@ -22,132 +22,158 @@ presence.on("UpdateData", async () => {
...
+
+	switch (true) {
+		case pathIncludes(path, "/home"):
+			presenceData.details = "Browsing...";
+			presenceData.state = "Viewing home page";
+			break;
+
+		case pathIncludes(path, "/search"):
+			presenceData.details = "Searching...";
+			presenceData.smallImageKey = "search";
```
Comment (DarkVillager) [inline websites/P/Paramount+/presence.ts:35]: Change these all to imgur links.

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/P/Paramount+/presence.ts [hunk 3]
```diff
@@ -22,132 +22,158 @@ presence.on("UpdateData", async () => {
...
-				title = jsonData.partOfSeries.name;
-
-				seasonEpi =
-					`S${jsonData.partOfSeason.seasonNumber}:` +
-					`E${jsonData.episodeNumber} ${jsonData.name}`;
+		{ pathname: path } = window.location;
+
+	switch (true) {
+		case pathIncludes(path, "/home"):
+			presenceData.details = "Browsing...";
```
Comment (theusaf) [inline websites/P/Paramount+/presence.ts:29]: Prefer strings without `...`
```suggestion
			presenceData.details = "Browsing";
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/P/Paramount+/presence.ts [hunk 4]
```diff
@@ -9,11 +9,18 @@ const presence = new Presence({
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
-let title: string,
-	seasonEpi: string,
-	movTitle: string,
-	vidMdTl: Element,
-	oldUrl: string;
+enum Assets {
```
Comment (DarkVillager) [inline websites/P/Paramount+/presence.ts:12]: remove

**Comment by DarkVillager** [comment]
@calicokatsu why close this?

**Comment by calicokatsu** [comment]
> @calicokatsu why close this?

I don't remember closing it. That's strange. I'll re-open it.

**Comment by Turtle-Boi** [comment]
how do i get this update

**Comment by calicokatsu** [comment]
@Turtle-Boi wait for #7360 to merge and it'll update it for you

**Pull request closed without merging**